src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -202,18 +202,35 @@ public PublicationController(PublicationService publicationService, ValidationSe
         return ResponseEntity.ok(publicationService.findAllByCourtId(searchValue, verification));
     }
 
+    @ApiResponses({
+        @ApiResponse(code = 200,
+            message = "Gets the artefact metadata",
+            response = Artefact.class),
+        @ApiResponse(code = 404,
+            message = "No artefact found matching given parameters and date requirements"),
+    })
+    @ApiOperation("GGets the metadata for the blob, given a specific artefact id")
+    @GetMapping("/{artefactId}")
+    public ResponseEntity<Artefact> getArtefactMetadata(
+        @PathVariable UUID artefactId, @RequestHeader Boolean verification) {
+
+        return ResponseEntity.ok(publicationService.getMetadataByArtefactId(artefactId, verification));
+
+    }
+
     @ApiResponses({
         @ApiResponse(code = 200,
             message = "Blob data from the given request in text format.",
             response = String.class),
         @ApiResponse(code = 404,
             message = "No artefact found matching given parameters and date requirements"),
     })
-    @ApiOperation("Get the info from within a blob given source artefact id and provenance")
-    @GetMapping("/{artefactId}")
-    public ResponseEntity<String> getBlobData(@PathVariable UUID artefactId, @RequestHeader Boolean verification) {
+    @ApiOperation("Gets the the payload for the blob, given a specific artefact ID")
+    @GetMapping("/{artefactId}/payload")
+    public ResponseEntity<String> getArtefactPayload(
```

Comment:
does this need to be the endpoint that returns string if its json but MultipartFile if its a flat file?

Reply:
or is that a diff ticket

Reply:
My vote would be a different ticket. I did have a quick look to see if it was a quick thing I could include as part of this, however I think it requires a think about extensions, file naming so thinking we leave it for now

Reply:
👍🏼  we'll need to make sure thats a captured ticket, could be able to extend this method to return either type when we get round to it